src/main/java/org/jitsi/jigasi/TranscriptionGatewaySession.java

```diff
@@ -259,6 +261,24 @@ void notifyChatRoomMemberLeft(ChatRoomMember chatMember)
         this.transcriber.participantLeft(identifier);
     }
 
+    @Override
+    void notifyChatRoomMemberUpdated(ChatRoomMember chatMember, Presence presence)
+    {
+        super.notifyChatRoomMemberUpdated(chatMember, presence);
+
+        TranslationLanguageExtension translationLanguage
+            = presence.getExtension(
+                TranslationLanguageExtension.ELEMENT_NAME,
+                TranslationLanguageExtension.NAMESPACE);
+
+        if(translationLanguage != null) {
+            String identifier = getParticipantIdentifier(chatMember);
+            String language = translationLanguage.getTranslationLanguage();
+
+            this.transcriber.updateParticipantLanguage(identifier, language);
+        }
```

Comment:
We need to handle the case where the user removes a language (by removing the extension from presence)